lib/taskana-core/src/test/java/acceptance/classification/UpdateClassificationAccTest.java

```diff
@@ -28,7 +28,7 @@
 import pro.taskana.exceptions.NotAuthorizedException;
 import pro.taskana.exceptions.TaskNotFoundException;
 import pro.taskana.impl.DaysToWorkingDaysConverter;
-import pro.taskana.impl.JobRunner;
+import pro.taskana.impl.jobs.JobRunner;
```

Comment:
leave JobRunner in the pro.taskana.impl package. This removes the necessity to make openConnection(), returnConnection() and getSqlSession() public.